### PR TITLE
Export more info to the test processes

### PR DESCRIPTION
**Pull request by mvollmer**
So that the test run itself might post review comments that contain
links to attached files.

### CODE REVIEW

File: s3-streamer [hunk 1]
```diff
@@ -295,7 +295,9 @@ def main():
         attachments_directory = AttachmentsDirectory(index, tmpdir)
         log_uploader = ChunkedUploader(index, 'log')
 
-        with subprocess.Popen(args.cmd, env=dict(os.environ, TEST_ATTACHMENTS=tmpdir),
+        with subprocess.Popen(args.cmd, env=dict(os.environ,
+                                                 TEST_ATTACHMENTS=tmpdir,
+                                                 LOGDIR_URL=destination.directory),
```
Comment (allisonkarlitskaya) [inline s3-streamer:300]: Let's call it `TEST_ATTACHMENTS_URL`, and please move this to a separate statement.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: s3-streamer (hunk 1 above)
Comment (mvollmer) [reply s3-streamer]: Done!

File: tests-invoke [hunk 2]
```diff
@@ -77,6 +77,9 @@ def main():
             entry_point = alt_entry_point
         cmd = ["timeout", "120m", entry_point]
 
+        os.environ["TEST_REVISION"] = opts.revision
```
Comment (martinpitt) [inline tests-invoke:80]: We already [do this](https://github.com/cockpit-project/cockpit/blob/main/test/common/run-tests#L528)

Comment (mvollmer) [reply tests-invoke:80]: But that's wrong half of the time, no? `HEAD` is the result of the rebase. We could use `ORIG_HEAD`, but that then assumes that a rebase has just been done, which might not be true.

I think it is best to set the values here, with authority.

Comment (martinpitt) [reply tests-invoke:80]: Ah, and you need the original HEAD from the proposed branch, without a rebase? AFAICS the only place where we use this variable so far is in bots/test-failure-policy, and there it doesn't matter much. OK then.

Comment (mvollmer) [reply tests-invoke:80]: I'd say the only reason to have this in the environment at all is to reliably point to the pre-rebase HEAD.  If you want HEAD, you can just look at it with `git rev-parse HEAD`.

**Comment by mvollmer** [comment]
> Thanks! This looks reasonable, but integration tests fail. Can you please re-submit his from origin? I'll then quickly ack/land it.

#3548

**Pull request closed without merging**